### DISCUSSION TITLE
v0.1.500

### DISCUSSION THREAD

**Discussion by DamianMorozov** [DamianMorozov/OpenTgResearcher#17] [Announcements]
### Added
- Proxy for downloads
- Client and proxy exception messages

<hr /><em>This discussion was created from the release <a href='https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.1.500'>v0.1.500</a>.</em>

**Comment by ryangar88** [comment]
Hi DamianMorozov,
Your tool is very helful, i tried but when this tool finish download the latest file, the tool not monitoring/listening for the next messages, so can you dev to after download latest file, the tool will continuos monitor/listening the next messages

**Reply by DamianMorozov** [reply]
Thank you for using the utility.
You're right, there is no automatic tracking of new messages. I may be able to start solving this problem in a couple of months.
